collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/ErrorDialog.java

```diff
@@ -24,6 +26,10 @@
 limitations under the License.
  */
 
+/**
+ * @deprecated Should be replaced with {@link MaterialAlertDialogFragment}
+ */
+@Deprecated
```

Comment:
I would have done the work to replace this instead of deprecating here, but wanted to avoid the extra risk given that this will be going into a hotfix.